### PR TITLE
TPCQC: adds PID cut configurables depends on AliceO2Group/AliceO2#9215

### PR DESCRIPTION
**Pull request by sbhawani**

### CODE REVIEW

**Comment by sbhawani** [comment]
This is based on the codes changes in O2 therefore can be merged only(to avoid CI error) after the merge of corresponding PR in O2 https://github.com/AliceO2Group/AliceO2/pull/9215

**Review by wiechula — APPROVED** [review]

**Comment by sbhawani** [comment]
Hi Jens this complete we can merge it

**Pull request merged**
